<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreevalWidget</class>
 <widget class="QWidget" name="TreevalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check tree from file</string>
  </property>
  <property name="windowIcon">
   <iconset theme="format-indent-more">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="rowWrapPolicy">
    <enum>QFormLayout::DontWrapRows</enum>
   </property>
   <property name="labelAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="formAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="pathLabel">
     <property name="text">
      <string>List source</string>
     </property>
     <property name="buddy">
      <cstring>pathButton</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="pathLayout">
     <item>
      <widget class="QLineEdit" name="pathEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Select source file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pathButton">
       <property name="text">
        <string>Open</string>
       </property>
       <property name="icon">
        <iconset theme="document-open">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="formatLabel">
     <property name="text">
      <string>Format</string>
     </property>
     <property name="buddy">
      <cstring>formatCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="formatCombo">
     <item>
      <property name="text">
       <string>Table</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>List</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="actionLabel">
     <property name="text">
      <string>Action</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="actionCombo">
     <item>
      <property name="text">
       <string>Check objects presence</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Deny object presence</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Force selected structure</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pathEdit</tabstop>
  <tabstop>pathButton</tabstop>
  <tabstop>formatCombo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
